### ISSUE TITLE
Support for alias_attributes

### ISSUE THREAD

**Issue by OskarEichler** [zdennis/activerecord-import#728]
Hey Zach,

First of all - amazing job with this gem! Our database has much fever anxiety attacks thanks to your work!

Secondly: When trying to perform a .import on a model with an alias_attribute, unfortunately we're getting an error message that the column does not exist. 

Would it be possible to add support for that? Thanks so much!

All the best,
Oskar

P.S. Example:

Model - YoutubeArtistRecord
Column - :source_artist_id
alias_attribute - :source_entity_id, :source_artist_id

When performing import with :source_artist_id everything works, but when using :source_entity_id we get an error.

**Comment by OskarEichler** [comment]
I think the implementation for this would be pretty quick, by doing the following:

```
alias_attributes = self.attribute_aliases.symbolize_keys.transform_values!(&:to_sym)
columns.map { |column| alias_attributes[column] || column}
```

**Comment by jkowens** [comment]
Hey @OskarEichler, sorry I just haven't had time to work on this project lately. If you can provide a PR that would be fantastic 👍

**Comment by jkowens** [comment]
Resolved by #799
